### DISCUSSION TITLE
imagemin/gulp-imagemin alternative?

### DISCUSSION THREAD

**Discussion by IanDelMar** [holger1411/understrap#1347] [General]
imagemin and hence gulp-imagemin are no longer maintained. 

>The Imagemin project is practically unmaintained.
>
>I started this project 7 years ago to scratch my own itch. I have not been actively involved in the project for many years. I have merged a pull requests once in a while because I felt bad about the project languishing, but I neither have the time or interest to dedicate what's needed to this project. We have tried to get other people involved. Some people have come and gone. But it's a thankless job and the project has a lot of problems.
>
>I would recommend using Squoosh instead, which actively maintained and also avoids a lot of the problems Imagemin are having with native binaries, as they're using WebAssembly.
>
>While I don't intend to deprecate the whole project yet. A lot of people still depend on it. I hope at least new users will consider using something else instead.

Source: https://github.com/imagemin/imagemin/issues/385

Can someone recommend an alternative?

**Comment by 0dp** [comment]
On the official github for imagemin all of the plugins are still maintained. So we can import them individually and make a new imagemin task; and remove imagemin-core.

Edit:
Supposedly, they are valid gulp tasks all by themselves

**Reply by IanDelMar** [reply]
I see. Then let's keep that in mind.

**Reply by 0dp** [reply]
Again squoosh does look like a more or less drop-in replacement. Worth a try since it would be much easier and cleaner.

Also must be totally honest. I've never used the imagemin task 🙈

For icons and stuff i normally use svgstore and build a sprite. For instance if i need any FA stuff i pick the SVG icons i need. Completely omitting the need for FA

**Reply by IanDelMar** [reply]
> Also must be totally honest. I've never used the imagemin task 🙈

Me neither 😆

**Reply by bacoords** [reply]
I've NEVER used the imagemin task. I'm working on re-doing the build process as a suggested PR. I don't plan on including imagemin unless someone else protests vehemently.

**Reply by 0dp** [reply]
If anything look i lnto how to cherry pick fa icons so to not include the whole shabang. There's a subsetter but it's been a mess since v5
